srcpkgs/findutils/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'findutils'
 pkgname=findutils
-version=4.9.0
+version=4.10.0
 revision=1
 bootstrap=yes
 build_style=gnu-configure
-configure_args="--program-prefix=g"
+configure_args="--program-prefix=g --disable-year2038"
```

Comment:
> On systems without any year2038 support configure with --disable-year2038.

Makes it sound like this shouldn't be done on systems that use 64 bit timestamps.